mteb/evaluation/MTEB.py

```diff
@@ -170,7 +175,28 @@ def _display_tasks(self, task_list, name=None):
 
     def mteb_benchmarks(self):
         """Get all benchmarks available in the MTEB."""
-        for benchmark in self._tasks:
+        from mteb.overview import MTEBTasks
+
+        # get all the MTEB specific benchmarks:
+        mteb_tasks = [t for t in self._tasks]
+
+        mteb_tasks, remaining_tasks = (
+            [t for t in self._tasks if "MTEB" in t.name],
+            [t for t in self._tasks if "MTEB" not in t.name],
+        )
```

Comment:
This is a bit inefficient; you can reduce runtime by half by iterating through it once